src/python/shorah_dec.py

```diff
@@ -25,6 +25,11 @@
     file of corrected reads
 '''
 
+from __future__ import print_function
+from __future__ import division
+from builtins import dict
+from future.utils import listitems
+from builtins import range
```

Comment:
I am a bit confused by the future/__future__ imports and by the fact that builtins module is already in the standard library.

My idea was to translate the code to python3 (with 2to3, I'd say). I see value in keeping python2 compatibility, but I would somehow make the switch more explicit.

To summarize, my idea was:
- shorah 1.0 runs in python 2.x and performs global analysis too,
- shorah 2.0 runs in python 3.x and only performs local analysis.

I understand that one more dependency (future) is not a big deal: after all this is taken care of by the package manager. But I would really make the switch, for us and for all, and say that, although back compatibility is offered, shorah 2.0 is written in python 3.